GermanFilter/sections/specific.txt

```diff
@@ -99,6 +99,7 @@ drwindows.de,wiesbadenaktuell.de,rauchmeldungen.de,simfans.de,wn.de,cosmopolitan
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+drwindows.de##li:has([class*="post"]:contains(Anzeige))
```

Comment:
Actually, not an ad, but removed it in the AdGuard Annoyances. Thanks for opening the pull request.